src/encoded/static/components/genome_browser.js

```diff
@@ -913,6 +919,7 @@ class GenomeBrowser extends React.Component {
                 labelLength += file.assay_term_name ? file.assay_term_name.length + 2 : 0;
                 labelLength += file.biosample_ontology && file.biosample_ontology.term_name ? file.biosample_ontology.term_name.length + 2 : 0;
                 labelLength += file.annotation_subtype ? file.annotation_subtype.length : 0;
+                labelLength += supplementalShortLabel ? supplementalShortLabel.length : 0;
```

Comment:
You can also do something like - 
`labelLength += supplementalShortLabel?.length || 0;`